### ISSUE TITLE
The question about LoRA structure after load from peft.

### ISSUE THREAD

**Issue by fxb392** [huggingface/peft#1322]
### Feature request

(up_proj): lora.Linear4bit(
              (base_layer): Linear4bit(in_features=4096, out_features=11008, bias=False)
              (lora_dropout): ModuleDict(
                (default): Dropout(p=0.05, inplace=False)
              )
              (lora_A): ModuleDict(
                (default): Linear(in_features=4096, out_features=8, bias=False)
              )
              (lora_B): ModuleDict(
                (default): Linear(in_features=8, out_features=11008, bias=False)
              )
              (lora_embedding_A): ParameterDict()
              (lora_embedding_B): ParameterDict()
            )
what are the role of lora_embedding_A and lora_embedding_B ？

**Comment by BenjaminBossan** [comment]
Those exist in case that you want to target `nn.Embedding` layers. As you can see, they are empty, which means that for your model, you didn't target any embeddings.
